plugins/routeros-suspension/src/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "1",
+  "information": {
+    "name": "routeros-suspension",
+    "displayName": "Mikrotik RouterOS - suspension",
+    "description": "Plugin for handling suspension on RouterOS devices.",
+    "url": "https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/routeros-suspension",
+    "version": "1.0.0",
+    "ucrmVersionCompliancy": {
```

Comment:
previous CR not resolved, completely remove ucrmVersionCompliancy, it's only compatible with UNMS